src/ASTInterpreter2.jl

```diff
@@ -354,6 +324,21 @@ function get_source(meth)
     end
 end
 
+function get_source(g::GeneratedFunctionStub)
+    b = g(g.argnames...)
```

Comment:
This doesn't seem quite right. Doesn't this take the argtypes (i.e. the `args = map(_Typeof, args)` at the callsite)?

Reply:
I see, this is used to step into the generator. I don't think this is quite right still then though, because calling `g(...)` will invoke the generator. What you want is to get the source of the generator to step into.